### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gotecq-annguyen**

### CODE REVIEW

**Review by gotecq-annguyen — COMMENTED** [review]
Should provide more content for that

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # Code Review Process
+
```
Comment (gotecq-annguyen) [inline README.md:2]: Add more content here

Comment (gotecq-annguyen) [reply README.md:2]: Oh, i will add now.

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum
```
Comment (ngquangan) [inline README.md:3]: @gotecq-annguyen  typo

**Review by ngquangan — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gotecq-annguyen) [reply README.md:3]: yeah, my mistake! I fixed

**Pull request merged**